modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -663,6 +663,21 @@ func (suite *KeeperTestSuite) TestQueryPacketCommitment() {
 			},
 			false,
 		},
+		{
+			"commitment not found",
```

Comment:
(reposting the comment here since it shows as outdated now due to some minor changes)

I added this test because the one above, called "channel not found" was not actually testing it in my understanding (since no check for HasChannel was performed), but it was actually hitting the line that checks for the commitment. So I created this to cover the scenario, by creating the channel and changing the Sequence parameter on the request, so that it passes the HasChannel check but fails after that.